dev_guide/deployments.adoc

```diff
@@ -93,11 +109,11 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
 You can start a new deployment manually using the web console, or from the CLI:
 
 ----
-$ oc deploy <deployment_config> --latest
+$ oc deploy --latest dc/<name>
 ----
 
 NOTE: If there's already a deployment in progress, the command will display a
-message and a new deployment will not be started.
+message and a new replication controller will not be deployed.
```

Comment:
@kargakis i'm looking forward to changing all this to replica sets if we eventually going to switch